tools/generator/src/Generator/AddBazelDependenciesTarget.swift

```diff
@@ -104,13 +106,41 @@ $(BAZEL_INTEGRATION_DIR)/calculate_output_groups.py
                 filePathResolver: filePathResolver
             )
 
+        var buildPhases = [
+            bazelBuildScript,
+            createLLDBSettingsModuleScript,
+        ]
+
+        if let preBuildScript = preBuildScript {
+            let scriptPath = try filePathResolver
+                .resolve(preBuildScript,
+                         forceAbsoluteProjectPath: true,
+                         mode: .script).string
+            let script = try createBuildScript(
+                in: pbxProj,
+                name: "Pre-build",
+                scriptPath: scriptPath
+            )
+            buildPhases.insert(script, at: 0)
+        }
+
+        if let postBuildScript = postBuildScript {
```

Comment:
Calling this post-build is not entirely correct—it's actually a post-bazel build.